### PR TITLE
Flamethrower fuels and rebalancing

### PR DESCRIPTION
**Pull request by John-Candlebury**
#### Summary

```SUMMARY: Content "Adds new ammo for flamethrowers, and rebalances range"```

#### Purpose of change

This adds new cheaper and craftable fuel types for flamethrowers. For all your chemistry-ignorant pyromanic characters. This also involved tweaking the flamethrower ranges to more closely resemble their real life numbers. 

#### Describe the solution

The range data was taken from M2 flamethrower technical manual, since the standard flamethrower is presumably an M2.  The document, in page 54, states a range of around 40 yards for napalm fuel (roughly 12 in game tiles) and 20 yards for liquid fuel. The stats of the base flamethrower were changed to reflect this, and the stats of the other versions were adjusted based on those numbers.

Flamethrower fuel gets its recipe from the M2 technical manual, which states, in page 36, that liquid fuel should be a 50/50 mix of gasoline and other less flammable oil. It suggests diesel among bunker and other heavy oils. It also seems to imply that exact compositions dont particularly matter.

Gelled gasoline gets its recipe from "The Anarchist Arsenal" which in page 7 has a recipe for napalm b, that involves dissolving polystyrene into gas.  I dont think the substance is exactly equal to napalm b, but checking with other sources, napalm b is primarily gasoline and polystyrene compounds, so the resultant substance is likely close enough to work out of a flamethrower.

Additionally I removed the spray can flamethrower, because the current implementation  didnt really make any sense whatsoever. 

#### Additional context

Heres the field manual/technical manual for the M2 flamethrower, https://archive.org/details/TM3-376A/page/n5

### CODE REVIEW

File: data/json/items/obsolete.json [hunk 1]
```diff
@@ -339,6 +339,23 @@
...
     "flags": [ "NEVER_JAMS", "INSTALL_DIFFICULT", "IRREMOVABLE" ]
   },
+  {
+    "id": "flamethrower_crude",
+    "copy-from": "flamethrower_base",
+    "type": "GUN",
+    "name": "spraycan flamethrower",
+    "description": "A favorite of hooligans around the world, this lighter duct-taped to a spray can is probably as dangerous to the user as anyone else.  The nozzle has been crudely modified to disperse gasoline, and a small pump attached to the side allows for repressurization, though this process takes a few minutes.",
+    "weight": 451,
+    "volume": 2,
```
Comment (anothersimulacrum) [inline data/json/items/obsolete.json]:
```suggestion
    "volume": "500 ml",
```

File: data/json/recipes/recipe_ammo.json [hunk 2]
```diff
@@ -873,6 +873,34 @@
...
+  {
+    "type": "recipe",
+    "result": "gelled_gasoline",
+    "category": "CC_AMMO",
+    "subcategory": "CSC_AMMO_OTHER",
+    "skill_used": "cooking",
+    "difficulty": 2,
+    "time": "60 m",
+    "qualities": [ { "id": "CONTAIN", "level": 1 } ],
+    "book_learn": [ [ "textbook_gaswarfare", 2 ], [ "textbook_anarch", 2, "Stuff THE MAN doesn't want you to know" ] ],
```
Comment (LaVeyanFiend) [inline data/json/recipes/recipe_ammo.json]: > "Stuff THE MAN doesn't want you to know"

Um, what does this do?

Comment (John-Candlebury) [reply data/json/recipes/recipe_ammo.json]: It hides the recipe from the book description, showing instead: "Stuff THE MAN doesn't want you to know"

**Comment by LaVeyanFiend** [comment]
Is cooking 2 not a weak requirement for this fuel?

**Comment by John-Candlebury** [comment]
> Is cooking 2 not too low a requirement for the fuels?

Not really, if you check the sources both recipes are pretty much get a container, eyeball stuff so gasoline and additive are roughly on half and half proportion.

Its hard to come up with the idea that you have to do this to get useful fuel, so both recipes are book locked, even though they are easy to perform.

**Pull request merged**